### DISCUSSION TITLE
👋 Welcome to the WEBMIDI.js Forum!

### DISCUSSION THREAD

**Discussion by djipco** [djipco/webmidi#154] [General]
## Welcome!

This is the official spot to interact with the WebMidi.js community. 

### Usage Guidelines

- **Post in the right category**. There are only 4, I'm sure you can handle it!
- **Respect others**. They are expected to do the same.
- **Be safe**. Remember that all posted content is public. Forever.
- **Be legit**. Hateful, illegal, spammy or copyright-infringing content will not be tolerated.

In short, be respectful, enjoy yourself, and help us build a great community. Cheers!

Jean-Philippe (a.k.a. djipco) 🤘﻿
